mysql-test/suite/galera/t/session_logbin_off.test

```diff
@@ -0,0 +1,76 @@
+--source include/galera_cluster.inc
+
+--connection node_1
+--let $assert_text = node_1 binlog should be enabled
+--let $sql_log_bin = query_get_value(SHOW VARIABLES LIKE 'sql_log_bin', Value, 1)
+--let $assert_cond = "$sql_log_bin" = "ON"
+--source include/assert.inc
+
+SHOW VARIABLES LIKE 'sql_log_bin';
+SHOW VARIABLES LIKE 'log_bin';
+
+CREATE TABLE t1 (a INT PRIMARY KEY);
+INSERT INTO t1 VALUES (0);
+
+SET SESSION sql_log_bin = OFF;
+
+INSERT INTO t1 VALUES (1);
+
+--connection node_2
+--let $assert_text = node_2 t1 should have two rows
+--let $assert_cond = [SELECT COUNT(*) FROM t1] = 2
+--source include/assert.inc
+
+--connection node_1
+DELETE FROM t1;
+
+--connection node_2
+--let $assert_text = node_2 t1 should have zero rows
```

Comment:
You may want to make this into a wait condition.  This assert may fail under load if the DELETE is not replicated in time.  Similar situation below also.

Reply:
Good point. I will rework in the same way for 5.7.